### PR TITLE
data:darvesh29

### PR DESCRIPTION
**Pull request by darvesh29**
## Changes proposed

Adding  a profile to the LinkFree account

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thanks for letting me create a profile on Linkfree

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3768"></a>

### CODE REVIEW

File: darvesh29.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Darvesh Chauhan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": 21 | Begginner in Open Source | Learning DevOps | Learning Web-Development | Java | DSA",
```
Comment (Pradumnasaraf) [inline darvesh29.json:5]:
```suggestion
  "bio": "21 | Begginner in Open Source | Learning DevOps | Learning Web-Development | Java | DSA",
```

File: darvesh29.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "avatar": "https://github.com/darvesh29.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/darveshere",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on Linkedin",
+      "url": https://www.linkedin.com/in/darveshchauhan/",
```
Comment (Pradumnasaraf) [inline darvesh29.json:15]:
```suggestion
      "url": "https://www.linkedin.com/in/darveshchauhan",
```

File: darvesh29.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/darvesh29",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Lets Connect on Insta",
+      "url": "https://www.instagram.com/shubham.ss0001/",
+      "icon": "FaInstagram
```
Comment (Pradumnasaraf) [inline darvesh29.json:26]:
```suggestion
      "icon": "FaInstagram"
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Add your name to the `data` folder and also fix the JSON syntax suggested. If you need help, feel free to ask.

**Comment by darvesh29** [comment]
> Hey,
> 
> Add your name to the `data` folder and also fix the JSON syntax suggested. If you need help, feel free to ask.

Hii Pradumansaraf , thanks for correcting me 
Where is the data folder?

**Comment by Pradumnasaraf** [comment]
> > Hey,
> > Add your name to the `data` folder and also fix the JSON syntax suggested. If you need help, feel free to ask.
> 
> Hii Pradumansaraf , thanks for correcting me Where is the data folder?

In the root directory - https://github.com/EddieHubCommunity/LinkFree/tree/main/data

**Comment by darvesh29** [comment]
> > > Hey,
> > > Add your name to the `data` folder and also fix the JSON syntax suggested. If you need help, feel free to ask.
> > 
> > 
> > Hii Pradumansaraf , thanks for correcting me Where is the data folder?
> 
> In the root directory - https://github.com/EddieHubCommunity/LinkFree/tree/main/data

i did it brother
when will this get merge now

**Comment by darvesh29** [comment]
> It is still at the root of the project not in the `data`. Please fix that.
> 
> if you need help feel free to ask

i am not understanding you
can you please provide me the steps to do this

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
To move your file into data folder, you just need to go to (https://github.com/darvesh29/LinkFree/edit/main/darvesh29.json). 
- And rename your file as `data/darvesh29.json`.
- Commit the changes, & it should be good to go then. 

Hope this helps. Feel free to ask if still confusion persists. :)

image

**Comment by darvesh29** [comment]
> To move your file into data folder, you just need to go to (https://github.com/darvesh29/LinkFree/edit/main/darvesh29.json).
> 
> * And rename your file as `data/darvesh29.json`.
> * Commit the changes, & it should be good to go then.
> 
> Hope this helps. Feel free to ask if still confusion persists. :)
> 
> image

hey !!!!Thank you
did it , is it good to go now?

**Pull request closed without merging**